tests/k8s-azure/manifest/cluster-api/cloud-config-vmss-short-cache-ttl.json

```diff
@@ -2,8 +2,7 @@
   "cloud": "AzurePublicCloud",
   "tenantId": "${AZURE_TENANT_ID}",
   "subscriptionId": "${AZURE_SUBSCRIPTION_ID}",
-  "aadClientId": "${AZURE_CLIENT_ID}",
-  "aadClientSecret": "${AZURE_CLIENT_SECRET}",
+  "userAssignedIdentityID": "${AZURE_CLIENT_ID_USER_ASSIGNED_IDENTITY}",
```

Comment:
This should make it so we can refer to this file as the default for CAPZ ci-entrypoint again and revert https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5023.